src/sentry/utils/sdk_crashes/sdk_crash_detection.py

```diff
@@ -32,10 +32,11 @@ def __init__(
     def detect_sdk_crash(
         self, event: Event, event_project_id: int, sample_rate: float
     ) -> Optional[Event]:
+
         should_detect_sdk_crash = (
             event.group
             and event.group.issue_category == GroupCategory.ERROR
-            and event.group.platform == "cocoa"
```

Comment:
Instead of the platform, we rely now on the SDK name in cocoa_sdk_crash_detector.